### PR TITLE
Provide curl_fpm inside fpm base image

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

File: Dockerfile.php_fpm [hunk 1]
```diff
@@ -12,6 +15,7 @@ RUN useradd -ms /bin/bash -G www-data elife && \
 ENV PATH=/srv/bin:${PATH}
 
 COPY scripts/ /root/scripts
+COPY utils/ /srv/bin/
```
Comment (thewilkybarkid) [inline Dockerfile.php_fpm]: Does this need `--chown=elife:elife`?

File: utils/curl_fpm [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+fi
+
+if [ -z "${PROJECT_FOLDER}" ]; then
+    echo "PROJECT_FOLDER environment variable must be set"
+    echo "Example: PROJECT_FOLDER=/srv/annotations"
+    exit 1
+fi
+
+path="$1"
+PHP_ENTRYPOINT="${PHP_ENTRYPOINT:-/web/app.php}"
```
Comment (thewilkybarkid) [inline utils/curl_fpm]: `/web/app.php` shouldn't be assumed (Symfony 4 changed to `/public/index.php` for its standard).

Comment (thewilkybarkid) [inline utils/curl_fpm]: Maybe make this a required variable?

Comment (giorgiosironi) [reply utils/curl_fpm]: Since it varies by application, the step from parameter with a default to parameter should be easy 👍

**Pull request merged**